README.md

```diff
@@ -107,60 +107,71 @@ CLI Options:
   -v, --version    Show the version
 
 Beautifier Options:
-  -s, --indent-size                 Indentation size [4]
-  -c, --indent-char                 Indentation character [" "]
-  -e, --eol                         character(s) to use as line terminators. (default newline - "\\n")');
+  -s, --indent-size             Indentation size [4]
```

Comment:
Oh whoops - I have no idea how that indentation changed.  I definitely didn't do that on purpose, I'll have to check to see if my editor is being smarter than it should be.

Sorry for that